src/_P009_MCP.ino

```diff
@@ -115,9 +115,10 @@ boolean Plugin_009(byte function, struct EventStruct *event, String& string)
         //parseString(string, 2) = command
         //parseString(string, 3) = gpio number
 
-        string.toLowerCase();
+        String tempstring = string;
+        tempstring.toLowerCase();
```

Comment:
I don't know how long these strings are, but you may also just use the first few chars using [substring()](https://www.arduino.cc/reference/en/language/variables/data-types/string/functions/substring/)

Something like `String tempstring(string.substring(0,10));`
You may have to play a little with the length .